src/plugins/dashboard/public/application/components/dashboard_editor.tsx

```diff
@@ -44,6 +47,14 @@ export const DashboardEditor = () => {
     appState,
   });
 
+  useEffect(() => {
+    if (showActionsInGroup) setHeaderVariant?.(HeaderVariant.APPLICATION);
```

Comment:
braces for single line if statements?

Reply:
We have strong linting rules that take care of these :/